### ISSUE TITLE
[Bug]: Changes to RS default energy cap in 1.1.0 don't apply to pre existing saves

### ISSUE THREAD

**Issue by nmills3** [AllTheMods/ATM-8#1804]
### Possible Fixes

Yes

### Modpack Version

1.1.0

### What happened?

If you updated from 1.0.28 to 1.1.0 the RS energy cap will still be 32000 unless you make a new save

### latest.log

https://gist.github.com/nmills3/3cc17fdecce44aa967fc687b7f9d9058

**Comment by Mitchell5200** [comment]
yes, that's due to the nature of the world specific serverconfig config folder, i did enable ServerConfig updater to fix that, but it seems to not have taken affect.
